src/components/ec/cuda/ec_cuda.c

```diff
@@ -73,6 +73,11 @@ static ucc_config_field_t ucc_ec_cuda_config_table[] = {
      ucc_offsetof(ucc_ec_cuda_config_t, reduce_num_blocks),
      UCC_CONFIG_TYPE_ULUNITS},
 
+    {"USE_COOPERATIVE_LAUNCH", "0",
+     "whether to use cooperative launch in persistent kernel executor",
+     ucc_offsetof(ucc_ec_cuda_config_t, use_cooperative_launch),
+     UCC_CONFIG_TYPE_UINT},
```

Comment:
UCC_CONFIG_TYPE_BOOL since it's either yes or no

Reply:
done. Now the default is 1